vehicles/default/models_municipal.json

```diff
@@ -2,1406 +2,2017 @@
     {
         "name": "100",
         "id": "ID_MARK_PISTENBULLY_MODEL_100",
-        "mark_id": "ID_MARK_PISTENBULLY"
+        "mark_id": "ID_MARK_PISTENBULLY",
+        "vehicle_type": "ID_TYPE_MUNICIPAL"
```

Comment:
No need to add `vehicle_type` field. All models already separated by type. In each file presents models for only one vihicle type.

Reply:
Remoced `vehicle_type` from all models files.